scripts/_init.bat

```diff
@@ -54,7 +54,7 @@ set vc_version=
 if "%vs_version%"=="VS16" (
     set cmake_generator=Visual Studio 16 2019
     set vsdir=vs16
-    set vc_version=vc16
+    set vc_version=vs16
```

Comment:
1. should we change also vc14 and vc15 below?
2. should we add also vs17 as in [libsnowflakeclient](https://github.com/snowflakedb/libsnowflakeclient/blob/master/scripts/_init.bat#L64)? and update the tested versions in [GH actions](https://github.com/snowflakedb/pdo_snowflake/blob/master/.github/workflows/build_test.yml#L38)?

Reply:
1. https://windows.php.net/downloads/php-sdk/deps/ only vs2019 changed to vs16. vc14 and vc15 are unchanged.
2. supporting vs17 is a big step I'd like to push that to next release.